### PR TITLE
update nav items

### PR DESCRIPTION
**Pull request by waqas1412**

### CODE REVIEW

File: src/app/containers/default-layout/_nav.ts [hunk 1]
```diff
@@ -12,233 +12,16 @@ export const navItems: INavData[] = [
...
+        url: '/resume/add'
       },
-      {
-        name: 'Error 404',
-        url: '/404'
-      },
-      {
-        name: 'Error 500',
-        url: '/500'
-      }
```
Comment (muhammad-aftab93) [inline src/app/containers/default-layout/_nav.ts:241]: @waqas1412 You should have these pages in your application.

File: src/app/containers/default-layout/_nav.ts [hunk 2]
```diff
@@ -12,233 +12,16 @@ export const navItems: INavData[] = [
...
-  },
-  {
-    name: 'Pages',
-    url: '/login',
-    iconComponent: { name: 'cil-star' },
-    children: [
-      {
-        name: 'Login',
-        url: '/login'
-      },
```
Comment (muhammad-aftab93) [inline src/app/containers/default-layout/_nav.ts:229]: There will be a login page as well. @waqas1412

**Review by muhammad-aftab93 — APPROVED** [review]

File: src/app/containers/default-layout/_nav.ts (hunk 1 above)
Comment (waqas1412) [reply src/app/containers/default-layout/_nav.ts:241]: I just updated the sidebar items, and I believe we do not need them to be in those items.

File: src/app/containers/default-layout/_nav.ts (hunk 2 above)
Comment (waqas1412) [reply src/app/containers/default-layout/_nav.ts:229]: These are not routes but sidebar items.

**Pull request merged**